libsentrykube/service.py

```diff
@@ -91,17 +90,85 @@ def get_service_value_overrides(
     if region_name == "saas":
         region_name = "us"
 
+    return service_regions_path / region_name
+
+
+def get_service_value_overrides(
+    service_name: str,
+    region_name: str,
+    cluster_name: str = "default",
+    external: bool = False,
+) -> dict:
+    """
+    For the given service, return the values specified in the corresponding _values.yaml.
+
+    If "external=True" is specified, treat the service name as the full service path.
+    """
     try:
-        service_override_file: Path = (
-            service_regions_path / region_name / f"{cluster_name}.yaml"
+        service_override_file = (
+            get_service_value_override_path(service_name, region_name, external)
+            / f"{cluster_name}.yaml"
         )
+
         with open(service_override_file, "rb") as f:
             values = yaml.safe_load(f)
     except FileNotFoundError:
         values = {}
     return values
 
 
+def get_managed_service_value_overrides(
```

Comment:
Nit: Maybe make this more explicit
```suggestion
def get_tools_managed_service_value_overrides(
```